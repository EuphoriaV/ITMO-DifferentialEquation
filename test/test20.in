1.76
(x + x)
2.11
4.09