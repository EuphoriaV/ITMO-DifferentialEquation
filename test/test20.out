0.1739802